### PR TITLE
Hackerrank Arrays excercises added-GabySFB

### PR DESCRIPTION
**Pull request by GabySFB**

### CODE REVIEW

File: GabrielaRamirez/EjerciciosHackerrank/src/hackkerank/day3/JavaArrayList.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class JavaArrayList {
+    public static void main(String[] args) {
+        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
+        Scanner in = new Scanner(System.in);
+        int lines = in.nextInt();
+
+        ArrayList[] set = new ArrayList[lines];
```
Comment (luisod78) [inline GabrielaRamirez/EjerciciosHackerrank/src/hackkerank/day3/JavaArrayList.java:12]: Code for interface not for implementation

File: GabrielaRamirez/EjerciciosHackerrank/src/hackkerank/day3/JavaArrayList.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                set[i].add(in.nextInt());
+            }
+        }
+        int queryNumber=in.nextInt();
+        for(int i=0;i<queryNumber;i++){
+            int position=in.nextInt();
+            int line=in.nextInt();
+            try{
+                System.out.println(set[position-1].get(line-1));
+            } catch(Exception e){
```
Comment (luisod78) [inline GabrielaRamirez/EjerciciosHackerrank/src/hackkerank/day3/JavaArrayList.java:26]: Prefer Specific Exceptions
https://dzone.com/articles/9-best-practices-to-handle-exceptions-in-java

File: GabrielaRamirez/EjerciciosHackerrank/src/hackkerank/day3/JavaHashset.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        int t = s.nextInt();
+        String [] pair_left = new String[t];
+        String [] pair_right = new String[t];
+
+        for (int i = 0; i < t; i++) {
+            pair_left[i] = s.next();
+            pair_right[i] = s.next();
+        }
+
+        HashSet<String> pairs = new HashSet<String>(t);
```
Comment (luisod78) [inline GabrielaRamirez/EjerciciosHackerrank/src/hackkerank/day3/JavaHashset.java:18]: Code for interface not for implementation

File: GabrielaRamirez/EjerciciosHackerrank/src/hackkerank/day3/ransomNote/MagazineNoteCheck.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        for (String word:magazine){
+
+            if(magazineWords.containsKey(word)){
+                int count = magazineWords.get(word);
+            }else{
+                magazineWords.put(word, 1);
+            }
+        }
+
+        for (String word:note){
```
Comment (luisod78) [inline GabrielaRamirez/EjerciciosHackerrank/src/hackkerank/day3/ransomNote/MagazineNoteCheck.java:20]: I see missing whitespaces in almost all your code.... please refer to:
https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace

File: GabrielaRamirez/EjerciciosHackerrank/src/hackkerank/day3/JavaMap.java [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+public class JavaMap {
+    public static void main(String []argh)
+    {
+        Scanner in = new Scanner(System.in);
+        int numberOfEntries=in.nextInt();
+        in.nextLine();
+        Map<String, Integer> map = new HashMap<>(numberOfEntries);
+
+        for(int i=0;i<numberOfEntries;i++)
```
Comment (luisod78) [inline GabrielaRamirez/EjerciciosHackerrank/src/hackkerank/day3/JavaMap.java:15]: I see missing whitespaces in almost all your code.... please refer to:
https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace

**Review by luisod78 — CHANGES_REQUESTED** [review]

**Review by globant-bootcamp — APPROVED** [review]

File: GabrielaRamirez/EjerciciosHackerrank/src/hackkerank/day3/JavaHashset.java [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+import java.util.Scanner;
+import java.util.Set;
+
+public class JavaHashset {
+
+    public static void main(String[] args) {
+
+        Scanner s = new Scanner(System.in);
+        int t = s.nextInt();
+        String[] pair_left = new String[t];
```
Comment (luisod78) [inline GabrielaRamirez/EjerciciosHackerrank/src/hackkerank/day3/JavaHashset.java:13]: snake case???
Java prefers CamelCase for var names
https://google.github.io/styleguide/javaguide.html#s5.3-camel-case

File: GabrielaRamirez/EjerciciosHackerrank/src/hackkerank/day3/ransomNote/MagazineNoteCheck.java (hunk 4 above)
Comment (luisod78) [reply GabrielaRamirez/EjerciciosHackerrank/src/hackkerank/day3/ransomNote/MagazineNoteCheck.java:20]: you need an space before each '{'
you need an space between ':'

In your others classes you did it right!!!!!

**Review by luisod78 — CHANGES_REQUESTED** [review]

**Review by luisod78 — APPROVED** [review]

**Pull request merged**